.gitignore

```diff
@@ -5,3 +5,7 @@
 /.synapse-base
 /synapse
 /var
+*~
```

Comment:
there's some attempt to sort this file; this and the below probably want to go alongside `.*.swp`